format/format.go

```diff
@@ -278,16 +282,25 @@ func (rf RestoreFlags) HasSpacesAroundBinaryOperationFlag() bool {
 	return rf.has(RestoreSpacesAroundBinaryOperation)
 }
 
+func (rf RestoreFlags) HasWithDefaultDB() bool {
+	return rf.has(RestoreWithDefaultDB)
+}
+
+func (rf RestoreFlags) HasStringWithoutDefaultCharset() bool {
+	return rf.has(RestoreStringWithoutDefaultCharset)
+}
```

Comment:
this doesn't seem to have any effect

Reply:
This is used in TiDB, related pr is [here](https://github.com/pingcap/tidb/pull/21275)